### PR TITLE
Initial implementation of ROS driver for fadecandy LED implementation

### PR DESCRIPTION
**Pull request by jonbinney**

### CODE REVIEW

**Comment by jonbinney** [comment]
@SaranyaRad could you review this?

**Comment by jonbinney** [comment]
@SaranyaRad actually let me add a readme before you review.

**Comment by jonbinney** [comment]
@SaranyaRad this is ready for review now

**Comment by jonbinney** [comment]
@turnerpeterk this PR has the driver code for the LEDs

File: scripts/example_client [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    bar_start = 0
+    bar_length = 3
+    num_leds = 21
+    color_i = 0
+    while not rospy.is_shutdown():
+        led_array = LEDArray()
+
+        for led_strip in range(8):
+            led_strip = LEDStrip()
```
Comment (srad14) [inline scripts/example_client]: It would make it clearer if you changed 8 to a descriptive constant as well.

**Review by srad14 — APPROVED** [review]
LGTM! Besides the small comment in the example file. Sorry for taking so long.

File: scripts/example_client (hunk 1 above)
Comment (jonbinney) [reply scripts/example_client]: Fixed, thanks!

**Pull request merged**